src/test/java/com/google/api/codegen/testdata/nodejs_main_library.baseline

```diff
@@ -126,34 +126,52 @@ function LibraryServiceApi(gaxGrpc, grpcClient, opts) {
       bundleDescriptors,
       {'x-goog-api-client': googleApiClient});
 
-  var stub = gaxGrpc.createStub(
-      servicePath,
-      port,
-      grpcClient.google.example.library.v1.LibraryService,
-      {sslCreds: sslCreds});
-  var methods = [
-    'createShelf',
-    'getShelf',
-    'listShelves',
-    'deleteShelf',
-    'mergeShelves',
-    'createBook',
-    'publishSeries',
-    'getBook',
-    'listBooks',
-    'deleteBook',
-    'updateBook',
-    'moveBook',
-    'listStrings',
-    'addComments',
-    'getBookFromArchive',
-    'updateBookIndex',
-    'addTag'
+  var libraryServiceStub = gaxGrpc.createStub(
+    servicePath,
+    port,
+    grpcClient.google.example.library.v1.LibraryService,
+    {sslCreds: sslCreds});
+  var libraryServiceStubMethods = [
+    'CreateShelf',
+    'GetShelf',
+    'ListShelves',
+    'DeleteShelf',
+    'MergeShelves',
+    'CreateBook',
+    'PublishSeries',
+    'GetBook',
+    'ListBooks',
+    'DeleteBook',
+    'UpdateBook',
+    'MoveBook',
+    'ListStrings',
+    'AddComments',
+    'GetBookFromArchive',
+    'UpdateBookIndex',
+    'AddTag'
```

Comment:
Those method names are lower-capital ones (as in the previous data) -- please fix.